extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -161,7 +161,7 @@ export function getElement(selector, el, selectionMethod) {
   } else if (selectionMethod == 'scope') {
     return el.parentElement.querySelector(selector);
   } else if (selector[0] == '#') {
-    return el.ownerDocument.getElementById(selector.slice(1));
+    return ampdoc.getElementById(selector.slice(1));
```

Comment:
I'd just do it here. And I'd simply check if ampdoc contains node and if it doesn't - force the value to `null`.